### ISSUE TITLE
Missing units (px, em) in generated CSS

### ISSUE THREAD

**Issue by wolfy1339** [idleberg/m8tro-bootstrap#2]
In various places in the generated CSS they is missing the unit of measurement. 
I believe you need to add it in the less files, specifically variables.less.
Example from .container-fluid: 
```css
.container-fluid {
margin-right: auto;
margin-left: auto;
padding-left: 10;
padding-right: 10;
}
```

**Comment by idleberg** [comment]
I think `container-fluid` is defined in Bootstrap itself and not in the LESS files of the theme. Maybe compiling with different LESS options will fix this.

**Comment by idleberg** [comment]
Fixed e607bf081e5d03b0a2121af6eca5dc6ea4d11ef9
